src/pybind/mgr/dashboard/run-frontend-unittests.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 set -e
-
+: ${CEPH_ROOT:=$PWD/../../../../}
```

Comment:
Does this always replace the value of `CEPH_ROOT`?

Reply:
Nope, it's a default value in case `$CEPH_ROOT` is not defined.